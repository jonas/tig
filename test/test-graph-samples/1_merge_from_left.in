commit A B C
    Commit A - Merge C into B
commit C B D
    Commit C - Merge D into B
commit B E
    Commit B after E
commit E D
    Commit E after D
commit D
    Commit D
