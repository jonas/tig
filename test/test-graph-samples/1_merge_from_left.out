●─╮ Commit A - Merge C into B
│ ●─╮ Commit C - Merge D into B
●─╯ │ Commit B after E
● ╭─╯ Commit E after D
◎─╯ Commit D
